docs/operational_guide/namespace_configuration.md

```diff
@@ -53,16 +53,16 @@ curl -X POST <M3_COORDINATOR_IP_ADDRESS>:<CONFIGURED_PORT(default 7201)>/api/v1/
     "snapshotEnabled": true,
     "repairEnabled": false,
     "retentionOptions": {
-      "retentionPeriodDuration": "2d",
-      "blockSizeDuration": "2h",
-      "bufferFutureDuration": "10m",
-      "bufferPastDuration": "10m",
+      "retentionPeriod": "2d",
+      "blockSize": "2h",
+      "bufferFuture": "10m",
+      "bufferPast": "10m",
       "blockDataExpiry": true,
-      "blockDataExpiryAfterNotAccessPeriodDuration": "5m"
+      "blockDataExpiryAfterNotAccessPeriod": "5m"
```

Comment:
This may have to be `blockDataExpiryAfterNotAccessedPeriod` [as here](https://sourcegraph.com/github.com/m3db/m3/-/blob/src/dbnode/retention/config.go#L35:18)

Reply:
@arnikola updated